tools/upgrade_versions.py

```diff
@@ -173,35 +173,36 @@ def _read_version_from_repo(self, repo_path):
             raise exception.PackageError(msg)
 
 
-def push_new_versions(versions_repo, release_date, versions_repo_push_url,
-                      versions_repo_push_branch, committer_name,
-                      committer_email):
+def commit_and_push_new_versions(versions_repo, release_date, versions_repo_push_url,
+                      versions_repo_push_branch, updater_name,
+                      updater_email, push):
```

Comment:
It was not already following it before. I'll use the opportunity to fix it anyway.